### ISSUE TITLE
Overline2 produced grouped data.frames

### ISSUE THREAD

**Issue by mem48** [ropensci/stplanr#316]
```
u = paste0(
  "https://github.com/npct/pct-outputs-regional-notR/raw/master/commute/msoa/",
  "isle-of-wight",
  "/rf.geojson"
)

sl = sf::read_sf(u)
rnet1 = overline2(sl, c("bicycle","dutch_slc"))
class(rnet1)
```

Should ungroup before returning results

**Comment by mem48** [comment]
Made a pull request

**Comment by Robinlovelace** [comment]
Fixed in #318
